.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Description
+<!-- Briefly describe noteworthy details -->
+
+## Type of change
+
+- [ ] *Bug fix (non-breaking change which fixes an issue)*
+- [ ] *New feature (non-breaking change which adds functionality)*
+- [ ] *Breaking change (fix or feature that would cause existing functionality to not work as expected)*
+- [ ] *Refactor* (existing code being refactored)
+- [ ] *This change requires a documentation update*
+
+# Submission Checklist
+
+- [ ] Build Verified
+- [ ] Bundle Verified
+- [ ] Lint passing
+- [ ] Unit tests passing
+- [ ] Integration tests passing
+
```

Comment:
Change logs need to be part of a submission. If we don't have it already, add a new help page that describes exactly all the steps involved here before a PR should be created.